pulpcore/pulpcore/constants.py

```diff
@@ -36,4 +37,22 @@
     (SYNC_MODES.MIRROR, 'Add new content and remove content is no longer in the remote repository.')
 )
 
+
+POLICY_MODE = SimpleNamespace(
```

Comment:
Suggest renaming to `DOWNLOAD_POLICY`.  Things are usually a _policy_ or a _mode_ so combining them seems both redundant and in conflict at the same time while not describing what the set of constants pertain to.

Reply:
Agree. fixed in next push.